plugins/Android/MAD_README.md

```diff
@@ -12,31 +12,46 @@ This is a custom unity webview fork to allow pausing and resuming unity while th
-   2. The script generates `WebViewPlugin.aar` and place it in `DEST_DIR` folder `../../build/Packager/Assets/Plugins/Android`
-5. If succeeded, 2 files (among others) would be generated - `WebViewPlugin.aar` and `core-1.6.0.aar.tmpl` under `build/Packager/Assets/Plugins/Android`. 
-
+   2. The script generates `WebViewPlugin-release.aar.tmpl` or `WebViewPlugin-development.aar.tmpl` and place it in `DEST_DIR` folder `../../build/Packager/Assets/Plugins/Android`
+5. If succeeded, 2 files (among others) would be generated - `WebViewPlugin-*.aar.tmpl` and `core-1.6.0.aar.tmpl` under `build/Packager/Assets/Plugins/Android`.
 
 If you get this error:
 > Failed to install the following Android SDK packages as some licences have not been accepted.
 > 
 Sometimes if you build android from unity editor it will generate the missing licences.
 If not working, you need to install the sdk tool follow this guide https://developer.android.com/studio/intro/update#sdk-manager in your android-studio/intelij.
 
+### Alternate solution to licensing issue
+You can install and update the appropriate Android SDKs directly into your Unity installation folder
+1. Go to Android Studio
+2. Go to Settings Menu > Android SDK
+3. in the Android SDK Location field (top) click Edit
+4. Navigate to your Unity install, e.g. `/Applications/Unity/Hub/Editor/2022.3.32f1/PlaybackEngines/AndroidPlayer/SDK`
+5. Update the SDKs there and accept the licenses
+
 
 ## Deploying new Plugin locally
-1. Copy `unity-webview` folder from `Packages` (Read only) and paste them in `Vendors` folderץ
+1. Copy `unity-webview` folder from `Packages` (Read only) and paste them in `Vendors` folder.
 2. Now you have 2 webviews and you can't build the project. Remove the unity-webview import from `packages.json`.
-3. You need to remove old assembly (will appear grey) and reimport new assembly (from `Vendors`) in `AssetsAssembly`.
-4. Now you can take `WebViewPlugin.aar` and `core-1.6.0.aar.tmpl` that were created in previous section and paste them in `Vendors/unity-webview/dist/package/Assets/Plugins/Android`.
+3. You need to remove old assembly (will appear greyed-out/error) and reimport new assembly (from `Vendors`) under `Assets/AssetsAssembly.asmdf`.
+4. Now you can take `WebViewPlugin-*.aar.tmpl` and `core-1.6.0.aar.tmpl` that were created in previous section and paste them in `Vendors/unity-webview/dist/package/Assets/Plugins/Android`.
 5. You can modify WebViewObject.cs as you like.
 6. If you add more changes to `CUnityPlayerActivity` or `CWebViewPlugin` Build the project, rebuild plugin and redo step 4.
  
 
 ## Deploying new Plugin in git
-1. Copy `WebViewPlugin.aar` and `core-1.6.0.aar.tmpl` from `build/Packager/Assets/Plugins/Android` to `dist/package/Assets/Plugins/Android`
-2. Push to git `WebViewPlugin.aar` and `core-1.6.0.aar.tmpl` from previous section.
```

Comment:
Why did you remove those steps?

Reply:
See updated version, the instructions are updated to include these again as I thought the project structure had changed.